nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/DomainDiscoveryService.java

```diff
@@ -46,35 +46,34 @@
 import com.sun.enterprise.config.serverbeans.Domain;
 import com.sun.enterprise.config.serverbeans.Node;
 import com.sun.enterprise.util.io.InstanceDirs;
+import static fish.payara.nucleus.hazelcast.MemberAddressPicker.initAddress;
```

Comment:
Too high up to let me suggest it, but the copyright needs updating to 2020 rather than 2018